config.md

```diff
@@ -304,46 +304,32 @@ For Windows based systems the user structure has the following fields:
 
 ## <a name="configPlatform" />Platform
 
-**`platform`** (object, REQUIRED) specifies the configuration's target platform.
-
-* **`os`** (string, REQUIRED) specifies the operating system family of the container configuration's specified [`root`](#root) file system bundle.
-  The runtime MUST generate an error if it does not support the specified **`os`**.
-  Bundles SHOULD use, and runtimes SHOULD understand, **`os`** entries listed in the Go Language document for [`GOOS`][go-environment].
```

Comment:
You'll want to drop [the reference at the ebd of this file][1] too, since you remove the only consumers.

[1]: https://github.com/opencontainers/runtime-spec/blame/e95b4f8ead3dd0a0ce2676339490f0ce126ca898/config.md#L859